src/output.rs

```diff
@@ -235,7 +239,10 @@ impl<'a> Readme<'a> {
 						newline(out, &indent)?;
 						newline(out, &indent)
 					},
-					Tag::List(_) => newline(out, &indent),
+					Tag::List(_) => {
+						debug_assert!(lists.pop_back().is_some());
```

Comment:
This pop here cleans up the `lists` vec when the list end. This is necessary for nested lists I think, and can save a slight amount of memory.

Reply:
Yes, this is probably necessary. That means it shouldn't be inside debug-only code though

Reply:
Ah, yes indeed.